guides/common/modules/proc_configuring-dhcpd-to-use-with-server.adoc

```diff
@@ -54,108 +61,139 @@ subnet _192.168.38.0_ netmask _255.255.255.0_ {
 }
 
 omapi-port 7911;
-key omapi_key {
+key _omapi_key_ {
 	algorithm hmac-md5;
-	secret "_My_Secret_";
+	secret "_<key_secret>_";
 };
-omapi-key omapi_key;
+omapi-key _omapi_key_;
 ----
 +
 Note that the `option routers` value is the IP address of your {ProjectServer} or {SmartProxyServer} that you want to use with an external DHCP service.
-. On {ProjectServer}, define each subnet.
-Do not set DHCP {SmartProxy} for the defined Subnet yet.
-+
-To prevent conflicts, set up the lease and reservation ranges separately.
-For example, if the lease range is 192.168.38.10 to 192.168.38.100, in the {ProjectWebUI} define the reservation range as 192.168.38.101 to 192.168.38.250.
-. Configure the firewall for external access to the DHCP server:
+
+. Perform the following steps on {ProjectServer}:
+
+
+.. Open the DHCP port in the `firewalld` service:
```

Comment:
The firewall must be opened on the DHCP server, but now it's listed under steps on `ProjectServer`.

Reply:
I moved it up.